go/errors.md

```diff
@@ -14,24 +14,18 @@ We use [microkit] to enrich **all** errors with stack traces.
 import "github.com/giantswarm/microerror"
 ```
 
-For errors returned by function call it is fine to just wrap them and return:
+While creating new error, then error from the same package (i.e. from error.go
+file) should be wrapped with `Maskf` with additional information. The
+additional information should be lowercased:
 
 ```go
-return microerror.Mask(err)
-```
+return microerror.Maskf(executionFailedError, "failed to request %#q", url)
```

Comment:
Why `%#q` instead of `%s`? It seems that this adds backquotes around the string. Is this for `luigi`?

Reply:
https://github.com/giantswarm/fmt/blob/master/go/log_messages.md#general-guidelines

Reply:
No this is rather for die-hard JSON readers so they don't see `\"` when using `#q`. And we want formatted strings quoted one way or another.